transforms/universal/pdf2parquet/python/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM docker.io/python:3.10.14-slim-bullseye
+
+RUN \
+    apt-get update \
+    # for opencv, towhee
+    && apt-get install -y libgl1 libglib2.0-0 curl wget \
+    && apt-get clean
+
+RUN pip install --upgrade pip 
+
+# install pytest
+RUN pip install --no-cache-dir pytest
+
+# Create a user and use it to run the transform
+RUN useradd -ms /bin/bash dpk
+USER dpk
+WORKDIR /home/dpk
+
+# Copy and install data processing libraries 
+# These are expected to be placed in the docker context before this is run (see the make image).
+COPY --chown=dpk:root data-processing-lib-python/ data-processing-lib-python/
+RUN cd data-processing-lib-python && pip install --no-cache-dir -e .
+
+# END OF STEPS destined for a data-prep-kit base image 
+
+COPY --chown=dpk:root pyproject.toml pyproject.toml 
+RUN pip install --no-cache-dir -e .
+
+# Download models
+RUN python -c 'from deepsearch_glm.utils.load_pretrained_models import load_pretrained_nlp_models; load_pretrained_nlp_models(verbose=True);'
+RUN python -c 'from docling.document_converter import DocumentConverter; s=DocumentConverter.download_models_hf(); print(f"Models cached in {s}")'
+
+# copy source data
+COPY --chown=dpk:root src/ .
+COPY ./src/pdf2parquet_local.py local/
```

Comment:
You need to copy in pdf2parquet_transform_python.py to be in the home dir.

Reply:
This is coming from the other examples e.g. https://github.com/IBM/data-prep-kit/blob/dev/transforms/universal/noop/python/Dockerfile (lines 20 and 28-30). All the files in `/src` are in the home directory (otherwise the tests won't even run).
If this extra `local/` is not needed we simply remove it.